packages/test/test-version-utils/src/compatConfig.ts

```diff
@@ -206,6 +209,30 @@ const genFullBackCompatConfig = (): CompatConfig[] => {
 	return _configList;
 };
 
+/**
+ * Returns true if compat test version is below the one provided as minimum version.
+ * It helps to filter lower verions configs that the ones intended to be tested on a
+ * particular suite.
+ * @param minVersion -
```

Comment:
These param descriptions are empty. You should either add a description or remove them.